app/models/book.rb

```diff
@@ -1,4 +1,15 @@
 class Book < ApplicationRecord
+  has_many :reviews
+
   validates :name, presence: true
   validates :description, allow_blank: true, presence: true
+
+  def summary_star
+    stars = 0.0
+    reviews.each do |review|
+      stars += review.star
+    end
+
+    stars / reviews.count
+  end
```

Comment:
I think this also works
```suggestion
  def summary_star
    stars = reviews.sum { |review| review.star }

    stars / reviews.count
  end
```